src/themes.js

```diff
@@ -1,4 +1,11 @@
-import { merge } from 'highcharts/highstock';
+import Highcharts from 'highcharts/highstock';
+
+function merge(a, b) {
+    if (process.env.NODE_ENV !== 'production') {
+        return a;
+    }
+    return Highcharts.merge(a, b);
+}
```

Comment:
Haha, nice.
We can probably extract these functions to a separate module.
We have similar code in the BinaryCharts.js:
if (Object.keys(Highcharts).length > 0) {